<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>hopin</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="login_wdg" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>480</width>
      <height>380</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QLabel" name="email_lbl">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Email</string>
     </property>
    </widget>
    <widget class="QLabel" name="pass_lbl">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Senha</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="email_txt">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>200</y>
       <width>331</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="pass_txt">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>250</y>
       <width>331</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="login_btn">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>320</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QWidget" name="logo_wdg" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>441</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#logo_wdg{
	background-color: rgb(204, 204, 204);
	border-radius: 8px;
	background-image: url(:/images/hopin);
	background-position: center;
	background-repeat: none;
}s</string>
     </property>
    </widget>
    <widget class="QLabel" name="loginwarn_lbl">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>320</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255,28,23);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="main_wdg" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>480</width>
      <height>380</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QPushButton" name="stop_btn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>320</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Parar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="start_btn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>320</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Iniciar</string>
     </property>
    </widget>
    <widget class="QLabel" name="idletime_lbl">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Tempo Ocioso</string>
     </property>
    </widget>
    <widget class="QSlider" name="idletime_sldr">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>170</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <number>2</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="sliderPosition">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="user_lbl">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Usu√°rio</string>
     </property>
    </widget>
    <widget class="QLabel" name="useremail_lbl">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="hopins_lbl">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Hopins</string>
     </property>
    </widget>
    <widget class="QLabel" name="hopinsvalue_lbl">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>90</y>
       <width>301</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>36</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(19, 214, 23);
background-color: rgb(255, 255, 255);
border: 0px solid white;
border-radius: 8px;</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QWidget" name="warn_wdg" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>441</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#warn_wdg{
	background-color: rgb(204, 204, 204);
	border-radius: 8px;
	background-image: url(:/images/alert);
	background-position: left;
	background-repeat: none;
}</string>
     </property>
     <widget class="QLabel" name="warn_lbl">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>0</y>
        <width>381</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Quantidade de tempo ocioso do sistema que o Hopin ir√° utilizar. A gera√ß√£o de hopins √© influenciada por este valor, ou seja, em 50% ir√° gerar metade da capacidade total.</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="percent_lbl">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>170</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>0 %</string>
     </property>
    </widget>
    <widget class="QLabel" name="status_lbl">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>320</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255,28,23);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <zorder>main_wdg</zorder>
   <zorder>login_wdg</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
